CHANGE-NOTES.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 
 ## v3.2.0
-- Added new `alt + enter` intention action to create a non-existing branch if user adds it to the machete file.
-- Added option to manually pick a commit for the fork point override.
+- Added a new `alt + enter` intention action to create a non-existing branch if user adds it to the machete file.
+- Added an option to manually pick a commit for the fork point override.
+- Fixed project-dependent settings persistence.
```

Comment:
@PawelLipski, isn't it a breaking change? 😕

Reply:
Theoretically it is... but it's so minuscule that I'd consider it a bugfix ;)